trendy-fashion/src/Components/Filter.jsx

```diff
@@ -1,365 +0,0 @@
-                    <Box as="span" flex='1' textAlign='left'>
-                      Discount
-                    </Box>
-                    <AccordionIcon />
-                  </AccordionButton>
-                </h2>
-                <AccordionPanel pb={4}>
-                  <div>
-                    <input type="checkbox" onChange={discountChangeHandler} value="0-20" />
-                    <label>0-20%</label>
-                  </div>
-                  <div>
-                    <input type="checkbox" onChange={discountChangeHandler} value="21-30" />
-                    <label >21-30%</label>
-                  </div>
-                  <div>
-                    <input type="checkbox" onChange={discountChangeHandler} value="31-40" />
-                    <label >31-40%</label>
-                  </div>
-                  <div>
-                    <input type="checkbox" onChange={discountChangeHandler} value="41-50" />
-                    <label>41-50%</label>
-                  </div>
-                  <div>
-                    <input type="checkbox" onChange={discountChangeHandler} value="51-60" />
-                    <label>51-60%</label>
-                  </div>
-                  <div >
-                    <input type="checkbox" onChange={discountChangeHandler} value="60-100" />
-                    <label > more than 60%</label>
-                  </div>
-                </AccordionPanel>
-              </AccordionItem>
-              <AccordionItem>
-                <h2>
-                  <AccordionButton>
-                    <Box as="span" flex='1' textAlign='left'>
-                      Brand
-                    </Box>
-                    <AccordionIcon />
-                  </AccordionButton>
-                </h2>
-                <AccordionPanel pb={4}>
-                  {
-                    menData.map((item, i) => {
-                      return (
-                        <div key={i}>
-                          <input type="checkbox"
-                            value={item}
-                            onChange={handleFilterCheckbox}
-                            checked={category.includes(item)}
-                          />
-                          <label>{item}</label>
-                        </div>
-                      )
-                    })
-                  }
-                </AccordionPanel>
-              </AccordionItem>
-            </Accordion>
-          </AccordionPanel>
-        </AccordionItem>
-        <AccordionItem>
-          <h2>
-            <AccordionButton>
-              <Box as="span" flex='1' textAlign='left'>
-                More Filters
-              </Box>
-              <AccordionIcon />
-            </AccordionButton>
-          </h2>
-          <AccordionPanel pb={4}>
-            Please select upto 3 categories to view more filters
-            <SeprateCategory />
-          </AccordionPanel>
-        </AccordionItem>
-      </Accordion>
-    </div>
-  )
-}
```

Comment:
this component is too big. Try to break this component into smaller components in different files.